### PR TITLE
backlog update and acrolinx updates

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
- **What's changed?** backlog update and acrolinx updates
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-368664](https://oktainc.atlassian.net/browse/OKTA-368664)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/concepts/api-access-management/index.md [hunk 1]
```diff
@@ -27,149 +27,153 @@ In general, use OpenID Connect to sign users in to apps, and use API Access Mana
...
+  * You can assign an OAuth 2.0 client to any number of authorization servers. Doing so allows you to generate various tokens, each with separate authorization policies, token expiration times, and scopes. The audience claim (`aud`) and client ID claim (`cid`) identify which token maps to which API product.
 
-* OAuth clients and authorization servers can be assigned on a many-to-many basis. This allows a developer to use a single OAuth client to retrieve access tokens from different authorization servers depending on the use case. The only difference is the endpoints accessed and the scopes requested.
+* You can assign OAuth clients and authorization servers on a many-to-many basis. This allows a developer to use a single OAuth client to retrieve access tokens from different authorization servers depending on the use case. The only difference is the endpoints accessed and the scopes requested.
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/concepts/api-access-management/index.md:146]:
```suggestion
* You can assign OAuth 2.0 clients and authorization servers on a many-to-many basis. This allows a developer to use a single OAuth 2.0 client to retrieve access tokens from different authorization servers depending on the use case. The only difference is the endpoints accessed and the scopes requested.
```

File: packages/@okta/vuepress-site/docs/concepts/api-access-management/index.md [hunk 2]
```diff
@@ -27,149 +27,153 @@ In general, use OpenID Connect to sign users in to apps, and use API Access Mana
...
+* A client (application) should never inspect the contents of an access token. For example, don't customize the client's UI based on scopes in the access token. The access token isn't meant for the client to read, it's meant for the client to use. However, there's still a large amount of metadata that Okta can attach to a token. For example, Okta may include metadata such as current validity, approved scopes, and information about the context in which Okta issues the token. Use the [token introspect endpoint](/docs/reference/api/oidc/#introspect) to access this data.
+* Use access tokens exclusively through an HTTP Authorization header instead of encoded into a payload or URL that may be logged or cached.
+* A client secret is a password and you should protect it. Therefore, don't embed access tokens in mobile applications, front-end JavaScript applications, or any other scenario where an attacker could access it.
+* Avoid using the Resource Owner Password grant type (`password`) except in legacy application or transitional scenarios. Okta recommends the authorization code, implicit, or hybrid grant types for most scenarios.
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/concepts/api-access-management/index.md:165]:
```suggestion
* Avoid using the Resource Owner Password grant type (`password`) except in legacy applications or transitional scenarios. Okta recommends the authorization code, implicit, or hybrid grant types for most scenarios.
```

Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/concepts/api-access-management/index.md:165]: Just a comment, I thought we weren't recommending "implicit" grant types anymore (?)

File: packages/@okta/vuepress-site/docs/concepts/api-access-management/index.md [hunk 3]
```diff
@@ -27,149 +27,153 @@ In general, use OpenID Connect to sign users in to apps, and use API Access Mana
...
+* For mobile and single-page web applications, using the Authorization Code with PKCE grant type is the best practice. The implicit or hybrid grant type is the next best option.
 * For Android or iOS applications, use [Okta Mobile SDK for Kotlin](https://github.com/okta/okta-mobile-kotlin) or [Okta Mobile SDK for Swift](https://github.com/okta/okta-mobile-swift).
-* When an application successfully validates an access token, cache the result until the expiration time (`exp`). Do this for validation that is either [local](/docs/guides/validate-access-tokens/) or via the [introspection endpoint](/docs/reference/api/oidc/#introspect).
+* When an application successfully validates an access token, cache the result until the expiration time (`exp`). Do this for validation that is either [local](/docs/guides/validate-access-tokens/) or through the [introspection endpoint](/docs/reference/api/oidc/#introspect).
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/concepts/api-access-management/index.md:168]:
```suggestion
* When an application successfully validates an access token, cache the result until the expiration time (`exp`). Do this for a validation that is either [local](/docs/guides/validate-access-tokens/) or through the [introspection endpoint](/docs/reference/api/oidc/#introspect).
```

**Review by brianduffield-okta — APPROVED** [review]
Few nits and one question.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/concepts/api-access-management/index.md](https://github.com/okta/okta-developer-docs/blob/ea77647fe58b6d327d502949fd1f4e24cd0d940f/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Fapi-access-management%2Findex.md) | [87](https://okta.acrolinx.cloud/api/v1/checking/scorecards/0ed7ae61-f1c8-4312-bfce-83943602a1c0) | 94  | 86  | 59  | 100  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.5096b5e3-df55-4fb2-be12-d648e1ee4fab?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/concepts/api-access-management/index.md](https://github.com/okta/okta-developer-docs/blob/58ccb277e5d5b7733f10604d4b1ca397cb306555/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Fapi-access-management%2Findex.md) | [87](https://okta.acrolinx.cloud/api/v1/checking/scorecards/2f6162cc-cdf2-4698-89a0-48335ede48f9) | 94  | 86  | 59  | 100  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.df7d4b20-54dd-4309-9f56-e3add8aa97ba?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/concepts/api-access-management/index.md](https://github.com/okta/okta-developer-docs/blob/b4a22530de863bb6e97fac978a3ddb981d11e44a/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Fapi-access-management%2Findex.md) | [87](https://okta.acrolinx.cloud/api/v1/checking/scorecards/da5b4b37-7a71-4ee6-8e6f-63515e99f1d7) | 94  | 86  | 59  | 100  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.afa57b44-7f50-4c04-8e56-c5d48dfd7356?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/concepts/api-access-management/index.md](https://github.com/okta/okta-developer-docs/blob/c3f80a6132deed01697aaf3c582efb25f3f2ef3d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Fapi-access-management%2Findex.md) | [87](https://okta.acrolinx.cloud/api/v1/checking/scorecards/0636130f-d282-47c2-b6da-abcc9ad1e740) | 94  | 86  | 59  | 100  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.3cee0ad5-54ed-4d88-95cc-2fbe70c25bc8?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/concepts/api-access-management/index.md](https://github.com/okta/okta-developer-docs/blob/15193692c14df76a285cc9056fc17e2e4b386997/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Fapi-access-management%2Findex.md) | [87](https://okta.acrolinx.cloud/api/v1/checking/scorecards/2e12f455-0c89-4c5a-9741-8c5a659dbafe) | 94  | 86  | 59  | 100  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.cd1453ca-0f78-4c74-b1db-5a57c210d469?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/concepts/api-access-management/index.md](https://github.com/okta/okta-developer-docs/blob/bc8b92cdbe8298b80d29309ccb8c871568eea386/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Fapi-access-management%2Findex.md) | [87](https://okta.acrolinx.cloud/api/v1/checking/scorecards/830a9b6e-fa84-4969-8f0c-a67260ff33ec) | 94  | 86  | 59  | 100  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.b0a79f22-f0e9-4594-84f8-b47aa1290c96?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Pull request merged**